gulpfile.js

```diff
@@ -26,7 +26,20 @@ const md = new Remarkable({
 });
 
 const packageJSON = require('./package.json');
-const vendorModules = Object.keys(packageJSON.dependencies);
+const excludedVendorModules = [
+  'babel-polyfill',
+  'l20n'
+];
+const includeVendorModules = [
+  'babel-polyfill/browser',
+  'l20n/dist/compat/web/l20n'
+];
+const vendorModules = Object.keys(packageJSON.dependencies)
+  .filter(name => {
+    if (excludedVendorModules.indexOf(name) !== -1) { return false; }
+    return true;
```

Comment:
Looks like `excludedVendorModules.indexOf(name) < 0` is actually what I need here, but that does work as a one-liner. I think I had it more complicated thinking I'd add more conditions, but YAGNI